src/components/common/ColonyTotalFunds/ColonyTotalFundsManageFunds.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+
+import Icon from '~shared/Icon';
+import Link from '~shared/Link';
+import { useColonyContext } from '~hooks';
+
+import styles from './ColonyTotalFunds.css';
+
+const MSG = defineMessages({
+  manageFundsLink: {
+    id: 'dashboard.ColonyTotalFundsManageFunds.manageFundsLink',
+    defaultMessage: 'Manage Funds',
+  },
+});
+
+const displayName = 'common.ColonyTotalFunds.ColonyTotalFundsManageFunds';
+
+const ColonyTotalFundsManageFunds = () => {
```

Comment:
Also think it'd be fine to call this `ManageFunds` or `ManageFundsLink` and drop the directory name from the component name